### PR TITLE
Functional Tests : BO - Design - Image Settings - Check stores image format

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Functional Tests : BO - Design - Image Settings - Check stores image format
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: & Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/4945370879
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+      if (!index) {
+        it('should go to BO', async function () {
+          await testContext.addContextItem(this, 'testIdentifier', `goToBoStores${arg.extension}`, baseContext);
+
+          await storePage.goToBO(page);
+
+          const pageTitle = await dashboardPage.getPageTitle(page);
+          await expect(pageTitle).to.contains(dashboardPage.pageTitle);
+        });
+        it('should go to \'Shop Parameters > Contact\' page', async function () {
```
Comment (nesrineabdmouleh) [inline tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts:300]:
```suggestion

        it('should go to \'Shop Parameters > Contact\' page', async function () {
```

File: tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+            page,
+            dashboardPage.shopParametersParentLink,
+            dashboardPage.contactLink,
+          );
+          await contactPage.closeSfToolBar(page);
+
+          const pageTitle = await contactPage.getPageTitle(page);
+          await expect(pageTitle).to.contains(contactPage.pageTitle);
+        });
+        it('should go to \'Stores\' tab', async function () {
```
Comment (nesrineabdmouleh) [inline tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts:314]:
```suggestion

        it('should go to \'Stores\' tab', async function () {
```

File: tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+          await testContext.addContextItem(this, 'testIdentifier', `goToStoresTab${arg.extension}`, baseContext);
+
+          await contactPage.goToStoresPage(page);
+
+          const pageTitle = await storesPage.getPageTitle(page);
+          await expect(pageTitle).to.contains(storesPage.pageTitle);
+        });
+      }
+      it('should filter list by name', async function () {
```
Comment (nesrineabdmouleh) [inline tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts:324]:
```suggestion

      it('should filter list by name', async function () {
```

File: tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts (hunk 2 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts:314]: Fixed

File: tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts (hunk 3 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/08_design/06_imageSettings/15_checkStoreImageFormat.ts:324]: Fixed

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @Progi1984

**Pull request merged**